### PR TITLE
Update README on issue reporting plus adding information on CLI locale problem

### PR DESCRIPTION
**Pull request by matsduf**
Clearified how to report issues and adding reference to missing CLI repository. Adding "notable bugs and issues" and in that adding information about problem with `--locale` option in CLI.

### CODE REVIEW

**Comment by mattias-p** [comment]
I think it'd be more helpful to link directly to the bug trackers, i.e. https://github.com/dotse/zonemaster-engine/issues etc.

File: README.md [hunk 1]
```diff
@@ -147,13 +147,35 @@ You can follow the project in these two mailinglists:
...
+
+
+## Notable bugs and issues
+
+### Selecting language with zonemaster-cli
+
+The command line tool `zonemaster-cli` has an option `--locale` with which you can select the language
+the output is written (currently English (the default), French, Swedish or Danish). If the environment
+variable `LANGUAGE` is set, then that might not work. The work-around is to unset the variable before
+running the `zonemaster-cli` command, e.g.
```
Comment (mattias-p) [inline README.md]: Is this still accurate? On FreeBSD the `--locale` parameter never works.

**Comment by matsduf** [comment]
I have done some updates to this PR. @sandoche2k, is it still OK? @mattias-p, is it OK now?

**Comment by mattias-p** [comment]
I will approve after these improvements are made:
1. Label the issues for the items listed under "Notable bugs and issues" as "Prio High".
2. Make sure that the titles of those issues are concise.
3. Make sure that the description texts for those issues are as clear as the text in each item.
4.  Update the "Notable bugs and issues" sections so that it's links to the issues instead of describing them.
5. Update the Release Process document with a step to maintain the "Notable bugs and issues" section.

As items from the "Notable bugs and issues" section are fixed in the develop branch, the issues should be clearly updated to state that they have been so.

We should probably also have a criteria for elevating issues to the "Notable bugs and issues" section.

Or preferably, make only updates 1, 2 and 3 above and remove the "Notable bugs and issues" section.

**Comment by matsduf** [comment]
@mattias-p, I think I have met your requests/requirements.

File: docs/internal-documentation/maintenance/ReleaseProcess.md [hunk 2]
```diff
@@ -118,22 +118,32 @@ tarballs:
...
 in this step become *accepted distribution tarballs*.
 
-## 9. Upload to CPAN
+## 9. Update Zonemaster repository main _README.md_
+
+If needed, update the following sections of the Zonemaster repository main _README.MD_ file:
+
+* Supported operating system versions
+* Supported database engine versions
+* Supported Perl versions
```
Comment (mattias-p) [inline docs/internal-documentation/maintenance/ReleaseProcess.md]: Updating supported versions for OSes, DB engines and Perl is already addressed in step 3 of this document. Please remove those items here.

**Pull request merged**